presto-docs/src/main/sphinx/optimizer/push-down.rst

```diff
@@ -0,0 +1,133 @@
+=========
+Push down
+=========
+
+Presto can push down processing queries, or segments of queries, down to the
+connected data source. This means that a specific function, or other operation,
+is passed through to the underlying database for processing. An improved overall
```

Comment:
I would also mention reduce network traffic and reduced load on remote database (in case of predicate pushdown for example)

Reply:
Rejigged to be a list of benefits